### DISCUSSION TITLE
4 Cameras D435i in sync approach

### DISCUSSION THREAD

**Discussion by afariach** [realsenseai/realsense-ros#2501] [Q&A]
Dear GitHub community,

For one of my projects we would like to synchronize  4 Intelrealsense D435i. This will be place on top of a robot, we want to sync them so we can have a full coverage, almost 360 degrees fov. I've read through various and is the only way to achieve this by connecting the realsense through the hw sync ports? And if that's the case, does a ros wrapped version for this sync exist? I looked an example on this [here](https://dev.intelrealsense.com/docs/multiple-depth-cameras-configuration) about merging point clouds and I was wondering as well if there was a code for this procedure. @MartyG-RealSense 

Thanks Everyone

**Comment by MartyG-RealSense** [comment]
Hi @afariach  Can you re-post this question at the RealSense ROS **Issues** section please?  Thanks!

https://github.com/IntelRealSense/realsense-ros/issues
